cmd/cloud/server.go

```diff
@@ -40,6 +40,7 @@ func init() {
 	serverCmd.PersistentFlags().String("public-subnets", "", "The public subnet IDs to use on AWS.")
 	serverCmd.PersistentFlags().Int("poll", 30, "The interval in seconds to poll for background work.")
 	serverCmd.PersistentFlags().Int("cluster-resource-threshold", 80, "The percent threshold where new installations won't be scheduled on a multi-tenant cluster")
+	serverCmd.PersistentFlags().Bool("keep-filestore-data", false, "Whether to preserve filestore data after installation deletion or not")
```

Comment:
Should we default to true for safety?

Reply:
Yeah, I wasn't sure if we wanted to err on the side of cleaning up by default or not and having to do it manually. Either one is fine with me.

Reply:
I'd lean towards not cleaning up by default